user/user_test.go

```diff
@@ -414,6 +414,72 @@ func Test_NewUserDetails_Validate_Valid(t *testing.T) {
 	}
 }
 
+func Test_NewUserDetails_HasRole_Multiple(t *testing.T) {
```

Comment:
as this is `HasRole_Multiple` do you think we should actually test it also has `other` ? just to make sure something isn't accidentally hard coded to `clinic`